src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -193,6 +193,7 @@
             "MaxPathsPerInvoke"
         ],
         "Bridged Device Basic Information": ["ProductAppearance"],
+        "Chime": ["InstalledChimeSounds"],
```

Comment:
Not needed; lists are automatically handled.

On the other hand, you probably _do_ want to add the two non-list attributes handled via AttributeAccessInterface here.

Reply:
fixed